### PR TITLE
Vendor in opencontainers/runc v1.0.0

### PR DESCRIPTION
**Pull request by rhatdan**
The RunningInUserNS Function was removed from libcontainer and is now
in contianerd.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,6 +6,7 @@ require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/Microsoft/go-winio v0.5.0
 	github.com/Microsoft/hcsshim v0.8.17
+	github.com/containerd/containerd v1.5.1
```
Comment (vrothberg) [inline go.mod:9]: I'd rather not add a dependency on `containerd`. Great code but also a big code base. Was `RunningInUserNS` removed from runc?

**Comment by rhatdan** [comment]
@giuseppe @saschagrunert @vrothberg @nalind @TomSweeneyRedHat PTAL

File: go.mod (hunk 1 above)
Comment (saschagrunert) [reply go.mod:9]: It think it's still available in 
```
	"github.com/opencontainers/runc/libcontainer/userns"
```

Comment (rhatdan) [reply go.mod:9]: Ok, maybe I missed it.

Comment (rhatdan) [reply go.mod:9]: I grepped through an older version of runc

**Review by vrothberg — APPROVED** [review]
LGTM, thanks @rhatdan

**Review by giuseppe — APPROVED** [review]
LGTM

**Pull request merged**